internal/pkg/log/assert.go

```diff
@@ -79,7 +80,8 @@ func valueMatches(value any, actualValue any) bool {
 
 // AssertJSONMessages checks that expected json messages appear in actual in the same order.
 // Actual string may have extra messages and the rest may have extra fields. String values are compared using wildcards.
-func AssertJSONMessages(t assert.TestingT, expected string, actual string, msgAndArgs ...any) bool {
+func AssertJSONMessages(t testing.TB, expected string, actual string, msgAndArgs ...any) bool {
+	t.Helper()
```

Comment:
Please fix `mockedT` 
![image](https://github.com/keboola/keboola-as-code/assets/19371734/f0e6c077-ada1-435f-bb25-5c1415416e09)

Reply:
Empty methods (with some `panic(not implemented)`)  will be fine, or not?